pgcontents/tests/test_pgmanager.py

```diff
@@ -121,6 +121,24 @@ def test_modified_date(self):
         renamed = cm.get(new_path)
         self.assertGreater(renamed['last_modified'], saved['last_modified'])
 
+    def test_get_file_id(self):
```

Comment:
The thing we care about the most is that ids persist across renames, right?  Can we test that case?